Update Remove-AppxPackageAutoUpdateSettings.md (#3536)
MicrosoftDocs/windows-powershell-docs: docset/winserver2022-ps/appx/Remove-AppxPackageAutoUpdateSettings.md

# Remove-AppxPackageAutoUpdateSettings

## SYNOPSIS

> Removes settings configured for a particular Windows App.

Reviewer: ```suggestion
Removes settings configured for a particular Windows app.
```

```

## DESCRIPTION

> The 'Remove-AppxPackageAutoUpdateSettings' PowerShell cmdlet removes the settings configured for a specific or all installed Windows Apps in relation to Auto Update and Repair.

Reviewer: ```suggestion
The 'Remove-AppxPackageAutoUpdateSettings' PowerShell cmdlet removes the settings configured for a specific or all installed Windows apps in relation to the Auto Update and Repair settings.
```

## EXAMPLES

### Example 1
```powershell
PS C:\> Remove-AppxPackageAutoUpdateSettings -PackageFullName publisher.package1_1.0.0.0_neutral__8wekyb3d8bbwe
```

> This will remove the Auto Update and Repair settings for a specific Windows App that has been installed and registered to the signed-in user.

Reviewer: ```suggestion
This example removes the Auto Update and Repair settings for a specific Windows app that has been installed and registered to the signed-in user.
```
